plugins/InspectionGadgets/InspectionGadgetsAnalysis/resources/messages/InspectionGadgetsBundle.properties

```diff
@@ -944,8 +944,8 @@ nested.try.statement.problem.descriptor=Nested <code>#ref</code> statement #loc
 exception.from.catch.which.doesnt.wrap.problem.descriptor=<code>#ref</code> inside 'catch' block ignores the caught exception #loc
 instanceof.catch.parameter.problem.descriptor='instanceof' on 'catch' parameter <code>#ref</code> #loc
 non.final.field.of.exception.problem.descriptor=Non-final field <code>#ref</code> of exception class #loc
-unnecessary.label.on.break.statement.problem.descriptor=Unnecessary label on <code>#ref</code> statement #loc
-unnecessary.label.on.continue.statement.problem.descriptor=Unnecessary label on <code>#ref</code> statement #loc
+unnecessary.label.on.break.statement.problem.descriptor=Unnecessary label <code>#ref</code> on break statement #loc
```

Comment:
ref should be replaced with the text of statement where the warning is attached and now it looks wrong.

Reply:
As only the label is unused (i.e. not the break statement), now we are highlighting only the label, not the break keyword. So now, `#ref` is only the label, not the whole statement. As a consequence, we have to rephrase the text. Is it OK?